src/data/boosts.js

```diff
@@ -238,17 +238,17 @@ export const CONTENT = {
       { href: 'https://disperse.app', label: 'Disperse App' },
     ],
   },
-  // SNAPSHOT: {
-  //   title: 'Snapshot Proposals',
-  //   description:
-  //     'Gasless voting for quicker, smaller decisions or just collecting signal.',
-  //   publisher: PUBLISHERS.DAOHAUS,
-  //   version: '0.5',
-  //   pars: ['AWAITING CONTENT'],
-  //   externalLinks: [
-  //     { href: 'https://discord.gg/daohaus', label: 'Boost Support' },
-  //   ],
-  // },
+  SNAPSHOT: {
+    title: 'Snapshot Proposals',
+    description:
+      'Gasless voting for quicker, smaller decisions or just collecting signal.',
+    publisher: PUBLISHERS.DAOHAUS,
+    version: '0.5',
+    pars: ['AWAITING CONTENT'],
```

Comment:
We have content for this now.

checkout this doc:
https://hackmd.io/o7X2OFPASpGGBWYpLEfOYw?view
![Screenshot from 2021-12-14 09-25-30](https://user-images.githubusercontent.com/6923345/146038530-09b028bc-6f1a-44d5-a171-21416592b140.png)